src/rf/rf/settings/base.py

```diff
@@ -55,6 +55,11 @@ def get_env_setting(setting):
 # END FILE STORAGE CONFIGURATION
 
 
+# STACK COLOR CONFIGURATION
+STACK_COLOR = environ.get('RF_STACK_COLOR', 'Black')
```

Comment:
I guess it's notable that this value is Black in development, but Blue/Green in staging/production.